zulipterminal/helper.py

```diff
@@ -493,3 +493,95 @@ def notify(title: str, html_text: str) -> None:
     if command:
         res = subprocess.run(shlex.split(command), stdout=subprocess.DEVNULL,
                              stderr=subprocess.STDOUT)
+
+
+def parse_html_table(table_element: Any) -> Tuple[List[str], List[List[str]]]:
+    """
+    Parses an HTML table to extract cell items and column alignments.
+    """
+    headers = table_element.thead.tr.find_all('th')
+    rows = table_element.tbody.find_all('tr')
+    alignments = []
+
+    # The table cells are stored row-wise:
+    # [[row0, row0, row0],
+    # [row1, row1, row1],
+    # [row2, row2, row2]].
+    # Add +1 to count the header row as well.
+    cells = [[] for _ in range(len(rows) + 1)]  # type: List[List[str]]
+
+    # Fill up the `cells` with the header/0th row and extract alignments.
+    for header in headers:
+        alignments.append(header.get(('align'), 'left'))
+        cells[0].append(header.text)
+
+    # Fill up the `cells` with body rows.
+    for index, row in enumerate(rows, start=1):
+        for tdata in row.find_all('td'):
+            cells[index].append(tdata.text)
+    return (alignments, cells)
+
+
+def unicode_border(lcorner: str, line: str, connector: str, rcorner: str,
```

Comment:
Is this more accurately a `unicode_row`? Or `row_with_only_borders`?

This would make it clearer when you interleave it with a `row_with_styled_content`, perhaps.

Reply:
I agree, these will be more accurate.